src/leaflike/migrations.clj

```diff
@@ -0,0 +1,23 @@
+(ns leaflike.migrations
+  (require [ragtime.jdbc :as jdbc]
+           [ragtime.repl :as repl]))
+
+(defn db-spec [] {:connection-uri "jdbc:postgresql://localhost:5432/leaflike"})
+
+(defn migration-config []
+  {:datastore  (jdbc/sql-database (db-spec))
+   :migrations (jdbc/load-resources "migrations")})
+
+(defn migrate-db []
+  (try
+    (let [out (with-out-str
```

Comment:
No point in capturing `stdout` unless you plan to put that output somewhere else. I'd suggest simply calling `repl/migrate`.
`stdout` is captured in `time-tracker`, but there's a reason for that. Take a look at what's going on there, and see if that use case is applicable here.